sdk/src/main/java/io/radar/sdk/RadarInAppMessageOperation.kt

```diff
@@ -0,0 +1,22 @@
+package io.radar.sdk
+
+/**
+ * Enum representing the possible operations for handling in-app messages.
+ * Used by RadarInAppMessageDelegate to control message display behavior.
+ */
+enum class RadarInAppMessageOperation {
+    /**
+     * Display the in-app message normally.
+     */
+    DISPLAY,
+    
+    /**
+     * Enqueue the in-app message to be displayed later.
+     */
+    ENQUEUE,
```

Comment:
we are not handling ENQUEUE right now, so I think we should just remove it to avoid confusion. We can add it back when we add that feature in v2+